<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page100</class>
 <widget class="QWidget" name="page100">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>page100</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#page100
{
background-color: rgb(0, 0, 55);
}

QLabel
{
background-color: rgb(0, 0, 55);
color: rgb(255, 255, 0);
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>41</y>
     <width>335</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Badge:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblElapsed">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>460</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblLoops">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="ATCMbutton" name="atcmButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>ATCMbutton widget</string>
   </property>
   <property name="pageName">
    <string notr="true">menu</string>
   </property>
   <property name="statusvar">
    <string notr="true"/>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
   <property name="pressText">
    <string>Menu</string>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="apparence">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time (ms):</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLen">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Loops:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Str Len:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ledReader">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ATCMlabel" name="atcmLabel_PLC_time">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>86400,00</string>
   </property>
   <property name="toolTip">
    <string>PLC_time</string>
   </property>
   <property name="whatsThis">
    <string>ATCMlabel widget</string>
   </property>
   <property name="variable">
    <string>PLC_time</string>
   </property>
   <property name="refresh">
    <number>200</number>
   </property>
   <property name="max">
    <string notr="true">0</string>
   </property>
   <property name="min">
    <string notr="true">0</string>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="bgColor">
    <color>
     <red>170</red>
     <green>170</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="fontColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 127);
color: rgb(255, 255, 255);
font: 9pt &quot;Ubuntu&quot;;
</string>
   </property>
   <property name="text">
    <string>Usb Tag Reader</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ATCMtime" name="atcmTime">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>ATCM QTime</string>
   </property>
   <property name="whatsThis">
    <string>ATCMtime widget</string>
   </property>
   <property name="bgColor">
    <color>
     <red>170</red>
     <green>170</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="fontColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
  </widget>
  <widget class="ATCMlabel" name="atcmLabel_HMI_Version">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20.001</string>
   </property>
   <property name="toolTip">
    <string>PLC_HMI_Version</string>
   </property>
   <property name="whatsThis">
    <string>ATCMlabel widget</string>
   </property>
   <property name="variable">
    <string>PLC_HMI_Version</string>
   </property>
   <property name="refresh">
    <number>200</number>
   </property>
   <property name="max">
    <string notr="true">0</string>
   </property>
   <property name="min">
    <string notr="true">0</string>
   </property>
   <property name="viewStatus">
    <bool>false</bool>
   </property>
   <property name="visibilityVar">
    <string/>
   </property>
   <property name="bgColor">
    <color>
     <red>170</red>
     <green>170</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="fontColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
  </widget>
  <widget class="ATCMdate" name="atcmDate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>ATCM QDate</string>
   </property>
   <property name="whatsThis">
    <string>ATCMdate widget</string>
   </property>
   <property name="bgColor">
    <color>
     <red>170</red>
     <green>170</green>
     <blue>127</blue>
    </color>
   </property>
   <property name="fontColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="borderWidth">
    <number>0</number>
   </property>
   <property name="borderRadius">
    <number>0</number>
   </property>
   <property name="format">
    <enum>ATCMdate::DD_MM_YYYY</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ATCMbutton</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMdate</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMlabel</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMtime</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>atcmButton</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page100</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goto_page(const char*,bool)</slot>
 </slots>
</ui>
